build/wordpress/nginx-php7-fpm/Dockerfile

```diff
@@ -80,18 +84,18 @@ RUN echo "installing WP-CLI" \
 # Nginx
 RUN apk --update add git
 
-RUN rm -rf /etc/nginx
-RUN git clone https://github.com/h5bp/server-configs-nginx /etc/nginx
+RUN rm -rf /etc/nginx && \
+    git clone https://github.com/h5bp/server-configs-nginx /etc/nginx && \
+    rm -rf /etc/nginx/sites-available && \
+    rm -rf /etc/nginx/sites-enabled && \
+    mkdir /etc/nginx/sites-enabled
```

Comment:
Reorganized everything down here just to reduce the number of intermediate layers we're creating